app/src/marketplace/components/ProductTypeChooser/index.jsx

```diff
@@ -14,79 +14,141 @@ import routes from '$routes'
 import docsLinks from '$shared/../docsLinks'
 import { productTypes } from '$mp/utils/constants'
 
-import styles from './productTypeChooser.pcss'
-
 type Props = {
     className?: string,
 }
 
+const Root = styled.div`
+  color: #323232;
+`
+
+const PageTitle = styled.div`
+    font-family: var(--sans);
+    font-size: 36px;
+    letter-spacing: 0;
+    line-height: 40px;
+    text-align: center;
+`
+
+const ProductChoices = styled.div`
+    margin-top: 6.25em;
+    display: grid;
+    grid-template-columns: 440px 440px;
+    grid-column-gap: 80px;
+`
+
+const Product = styled.div`
+    display: grid;
+    grid-row-gap: 1px;
+
+    > *:first-child {
+        border-top-left-radius: 6px;
+        border-top-right-radius: 6px;
+    }
+
+    > *:last-child {
+        border-bottom-left-radius: 6px;
+        border-bottom-right-radius: 6px;
+    }
+
+    > * {
+        background: white;
+    }
+`
+
+const ProductTitle = styled(Translate)`
+    height: 80px;
+    line-height: 80px;
+    text-align: center;
+    font-size: 20px;
+    letter-spacing: 0;
+    font-family: var(--sans);
+`
+
+const ProductImage = styled.div`
+    height: 304px;
+    position: relative;
+
+    > img {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        height: 200px;
+        width: auto;
+    }
+`
+
+const ProductText = styled.div`
+    padding: 1.875rem 3.15rem 2.5rem 3.15rem;
```

Comment:
That's an interesting range of padding, heh. Also, `1.875rem 3.15rem 2.5rem` should be fine. Fine to leave it tho.